esmvaltool/cmorizers/obs/cmor_config/WOA.yml

```diff
@@ -15,29 +15,39 @@ variables:
   thetao:
     mip: Omon
     raw: t_an
-    file: woa13_decav81B0_t
+    file: woa18_decav_t00_01.nc
+    #file: woa13_decav81B0_t
   so:
     mip: Omon
     raw: s_an
-    file: woa13_decav81B0_s
-  o2:
-    mip: Oyr
-    raw: o_an
-    file: woa13_all_o
-  no3:
-    mip: Oyr
-    raw: n_an
-    file: woa13_all_n
-  po4:
-    mip: Oyr
-    raw: p_an
-    file: woa13_all_p
-  si:
-    mip: Oyr
-    raw: i_an
-    file: woa13_all_i
+    file: woa18_decav_s00_01.nc
+    #file: woa13_decav81B0_s
+  tos:
+    mip: Omon
+    raw: t_an
+    file: woa18_decav_t00_01.nc
+  sos:
+    mip: Omon
+    raw: s_an
+    file: woa18_decav_s00_01.nc
+#  o2:
+#    mip: Oyr
+#    raw: o_an
+#    file: woa13_all_o
+#  no3:
+#    mip: Oyr
+#    raw: n_an
+#    file: woa13_all_n
+#  po4:
+#    mip: Oyr
+#    raw: p_an
+#    file: woa13_all_p
+#  si:
+#    mip: Oyr
+#    raw: i_an
+#    file: woa13_all_i
```

Comment:
what's the deal with these commented out ones? Can I remove them?

Reply:
You could here also remove the commented-out sign.

Reply:
done, cheers, Lisa!